reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-73.md

```diff
@@ -10,29 +10,36 @@ PowerShell 7.3 is the next preview release, built on .NET 6.0.
 
 PowerShell 7.3-preview.1 includes the following features, updates, and breaking changes.
 
+Improved error handling
+
 - Set `$?` correctly for command expression with redirections (#16046)
 - Fix a casting error when using `$PSNativeCommandUseErrorActionPreference` (#15993)
 - Make the native command error handling optionally honor `ErrorActionPreference` (#15897)
+- Specify the executable path as `TargetObject` for non-zero exit code ErrorRecord (#16108) (Thanks
+
+Tab completion improvements
+
 - Fix tab completion within the script block specified for the `ValidateScriptAttribute`. (#14550)
   (Thanks @MartinGC94!)
+- Added tab completion for loop labels after Break/Continue (#16438) (Thanks @MartinGC94!)
+
+Updated cmdlets
+
 - Add `-HttpVersion` parameter to web cmdlets (#15853) (Thanks @hayhay27!)
 - Add support to web cmdlets for open-ended input tags (#16193) (Thanks @farmerau!)
 - Fix `ConvertTo-Json -Depth` to allow 100 at maximum (#16197) (Thanks @KevRitchie!)
-- Specify the executable path as `TargetObject` for non-zero exit code ErrorRecord (#16108) (Thanks
   @rkeithhill!)
-- Invoke-Command: improve handling of variables with $using: expression (#16113) (Thanks @dwtaber!)
+- Invoke-Command: improve handling of variables with `$using`: expression (#16113) (Thanks @dwtaber!)
+- Add -StrictMode to Invoke-Command to allow specifying strict mode when invoking command locally
 - Add `clean` block to script block as a peer to `begin`, `process`, and `end` to allow easy
   resource cleanup (#15177)
+  (#16545) (Thanks @Thomas-Yu!)
+- Add support for OpenSSH options for PSRP over SSH commands (#12802) (Thanks @BrannenGH!)
```

Comment:
Attempting to remove the double "for" here; I _think_ this still captures what was changed. Not sure if "PSRP over SSH commands" should be expanded to the full list or not, leaning towards not.
```suggestion
- Add `-Options` to the PSRP over SSH commands to allow passing OpenSSH options directly (#12802) (Thanks @BrannenGH!)
```